packages/repository/test/unit/decorator/model-and-relation.ts

```diff
@@ -70,6 +85,8 @@ describe('model decorator', () => {
 
     // Validates that property no longer requires a parameter
     @property() isShipped: boolean;
+
+    @array(Product) items: Product[];
```

Comment:
We should validate to make sure the decorated property has design-time type to be `Array` even the item type is not captured by TypeScript at the moment.